train.py

```diff
@@ -470,7 +472,8 @@ def train(hyp, opt, device, tb_writer=None):
                 'obj_pw': (1, 0.5, 2.0),  # obj BCELoss positive_weight
                 'iou_t': (0, 0.1, 0.7),  # IoU training threshold
                 'anchor_t': (1, 2.0, 8.0),  # anchor-multiple threshold
-                'anchors': (2, 2.0, 10.0),  # anchors per output grid (0 to ignore)
```

Comment:
Actually the anchors parameter is fully functional now, hyp evolution can evolve this just like any other parameter. I recently evolved a custom dataset where the anchor count evolved upwards to almost 8 anchors (from the default 3).